src/components/ProgressCircle/ProgressCircle.tsx

```diff
@@ -0,0 +1,101 @@
+type ProgressCircleProps = {
+	percent: number;
+	value: number;
+	colorVariant: keyof typeof COLOR_VARIANTS;
+	transparentVariant: keyof typeof TRANSPARENT_VARIANTS;
+	label: string;
+};
+
+const TRANSPARENT_VARIANTS = {
+	1: {
+		opacity: "0.5",
+	},
+	2: {
+		opacity: "1",
+	},
+};
+
+const COLOR_VARIANTS = {
+	1: {
+		stroke: "stroke-blue-700",
+	},
+	2: {
+		stroke: "stroke-blue-500",
+	},
+	3: {
+		stroke: "stroke-teal-300",
+	},
+};
+
+export function ProgressCircle({
+	percent,
+	value,
+	colorVariant,
+	transparentVariant,
+	label,
+}: ProgressCircleProps) {
+	const circleRadius = 60;
```

Comment:
All of static values which does not require props for calculation should be outside of component and uppercase

Reply:
done